src/olympia/accounts/tests/test_views.py

```diff
@@ -1268,7 +1268,28 @@ def test_view_deleted(self):
         response = self.client.get(self.url)
         assert response.status_code == 404
 
-    def test_is_not_full_public_profile_because_not_developer(self):
+    def test_is_not_full_public_profile_because_not_developer_but_fields_present(self):
```

Comment:
Nit: I would rather this be two different tests, but since we are trying to cherry-pick no need.

Reply:
It is two tests:
- `test_is_not_full_public_profile_because_not_developer_but_fields_present` tests the case with the shim
- `test_is_not_full_public_profile_because_not_developer_no_fields` tests without the shim is, but as it's identical to the original `test_is_not_full_public_profile_because_not_developer` it doesn't show any changed lines

Reply:
I mean the 2 sets of assertions in the first test.